spec/storage/file_spec.rb

```diff
@@ -72,19 +72,19 @@
     after { FileUtils.rm_rf(cache_dir) }
 
     it "clears all files older than now in the default cache directory" do
-      Timecop.freeze(today) { uploader_class.clean_cached_files!(0) }
+      Timecop.freeze(today) { storage.clean_cache!(0) }
```

Comment:
Please don't change these calls for `#clean_cached_files!`. This should be considered as a public API which needs to be tested.

To allow it to be executed for a specific uploader class, you'll need this change.

https://github.com/carrierwaveuploader/carrierwave/blob/a67bfb696dcba14c7cdfa2c1b5481f04d3ef8dae/lib/carrierwave/uploader/cache.rb#L68

```diff
-          (cache_storage || storage).new(CarrierWave::Uploader::Base.new).clean_cache!(seconds)
+          (cache_storage || storage).new(new).clean_cache!(seconds)
```